lib/Jsrt/Core/JsrtCore.cpp

```diff
@@ -868,10 +868,15 @@ static void CastCopy(const SrcChar* src, DstChar* dst, size_t count)
 }
 
 CHAKRA_API JsCreateString(
-    _In_ const char *content,
+    _In_opt_ const char *content,
     _In_ size_t length,
     _Out_ JsValueRef *value)
 {
+	if (length == 0)
+	{
+		content = "";
+	}
```

Comment:
Please could you bring this in line with the way you've done JsPointerToString below? If length = 0, skip the `PARAM_NOT_NULL(content);` and then use `scriptContext->GetLibrary()->GetEmptyString();` instead of the logic on lines 896 and 898 below.

Reply:
Oh, I see. I had issues with GetEmptyString() because I was including line 898. Executing 896 and 898 them when length isn't zero fixed this. Thank you.